gprofiler/main.py

```diff
@@ -300,6 +301,7 @@ def _snapshot(self) -> None:
                 self._enrichment_options,
                 metadata,
                 metrics,
+                self._profiling_mode,  # For now just send the same profiling mode from the args.
```

Comment:
You have it in the metadata already (via `run_args`), don't need to pass it again

Reply:
It needs an explicit field in `metadata`, don't pass it here as a lone field

Reply:
I passed it because in the future we want to send multiple profiling modes for the same profiler instance, do you think I should add it to `metadata` before the call?

Reply:
Ok added to `metadata` prior the call